locust/test/test_main.py

```diff
@@ -191,6 +192,58 @@ def my_task(self):
             self.assertIn("Shutting down (exit code 0)", stderr)
             self.assertEqual(0, proc.returncode)
 
+    def test_percentile_parameter(self):
+        port = get_free_tcp_port()
+        with temporary_file(
+            content=textwrap.dedent(
+                """
+            from locust import User, task, constant, events
+            from locust.stats import PERCENTILES_TO_CHART
+            PERCENTILES_TO_CHART[0] = 0.9
+            PERCENTILES_TO_CHART[1] = 0.4
+            class TestUser(User):
+                wait_time = constant(3)
+                @task
+                def my_task(self):
+                    print("running my_task()")
+        """
+            )
+        ) as file_path:
+            proc = subprocess.Popen(
+                ["locust", "-f", file_path, "--web-port", str(port), "--autostart"], stdout=PIPE, stderr=PIPE, text=True
+            )
+            gevent.sleep(1)
+            proc.send_signal(signal.SIGTERM)
+            stdout, stderr = proc.communicate()
+            self.assertIn("Starting web interface at", stderr)
+            try:
+                response = requests.get(f"http://0.0.0.0:{port}/")
+                self.assertEqual(200, response.status_code)
+            except Exception:
+                pass
+
+    def test_invalid_percentile_parameter(self):
+        with temporary_file(
+            content=textwrap.dedent(
+                """
+            from locust import User, task, constant, events
+            from locust.stats import PERCENTILES_TO_CHART
+            PERCENTILES_TO_CHART[0] = 1.2
+            class TestUser(User):
+                wait_time = constant(3)
+                @task
+                def my_task(self):
+                    print("running my_task()")
+        """
+            )
+        ) as file_path:
+            proc = subprocess.Popen(["locust", "-f", file_path, "--autostart"], stdout=PIPE, stderr=PIPE, text=True)
+            gevent.sleep(1)
+            proc.send_signal(signal.SIGTERM)
```

Comment:
No need to send signal, I think you can remove it.

Reply:
Will check